backend/src/services/model.ts

```diff
@@ -335,6 +342,39 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   return model
 }
 
+async function checkCollaboratorAuthorisation(collaborators: CollaboratorEntry[]) {
```

Comment:
Could you change the name of this to `validateCollaborators` as this function isn't related to authorisation?